manager/integration/tests/common.py

```diff
@@ -4284,9 +4296,15 @@ def wait_for_rebuild_start(client, volume_name,
     assert started
     return status.fromReplica, status.replica
 
+def wait_for_restoration_start(client, name):
+    return wait_for_volume_status(client, name,
+                                  VOLUME_FIELD_RESTOREINITIATED,
+                                  True)
+
 
 def wait_for_volume_restoration_completed(client, name):
     wait_for_volume_creation(client, name)
+    wait_for_restoration_start(client,name)
```

Comment:
Would this cause the test cases using this function become flaky?

Reply:
It makes the test more stable because we check the restoreInitiated field which is set only once in the volume lifetime.